README.rst

```diff
@@ -56,8 +56,7 @@ First, create a ``conf.py`` file in your project's directory, next to
             description="Used for cryptographic signing. "
             "https://docs.djangoproject.com/en/2.0/ref/settings/#secret-key")
 
-        class Config:
-            default_files = ["/etc/myproject/myproject.yaml", "myproject.yaml"]
+        model_config = {"default_files": ["/etc/myproject/myproject.yaml", "myproject.yaml"]}
```

Comment:
The readme still mentions `Config` below, should get updated to `model_config` as well.